recipes/confu_json/all/test_package/conanfile.py

```diff
@@ -6,6 +6,9 @@ class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     generators = "cmake"
 
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
```

Comment:
```suggestion
```
Only if your test package is ported to Conan v2. For instance: https://github.com/conan-io/conan-center-index/pull/12458/files#diff-27f9a641a3f432a849fa0cd6294d8b562be49b5dab88b13b922a0d0d72d2fb94